### PR TITLE
Game logic

### PR DESCRIPTION
**Pull request by jacquelyn-bradshaw**

### CODE REVIEW

File: scripts/app.js [hunk 1]
```diff
@@ -23,10 +33,30 @@ const formElement = document.getElementById("config-form");
...
+const playerNameAlertModal = document.getElementById("player-name-alert");
+
+const playerTurnElement = document.getElementById("active-player-name-element");
+
+const gameBoardElements = document.querySelectorAll("#game-board-element li");
+
+const gameOverArticle = document.getElementById("game-over-element");
+const gameOverHeading = document.getElementById("game-over-heading");
+
+//config event listeners
```
Comment (johnbradshw) [inline scripts/app.js:29]: Do you want to create another branch and have a go at adding all these event listeners directly in the html?

File: scripts/game.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  }
+
+  resetGameStatus();
+
+  activeGameSection.style.display = "block";
+  playerTurnElement.textContent = players[activePlayer].name;
+}
+
+function switchPlayer() {
+  if (activePlayer === 0) {
```
Comment (johnbradshw) [inline scripts/game.js]: Can you explain what the difference between `==` and `===` is?

Comment (johnbradshw) [inline scripts/game.js]: Have you learnt about ternary operators yet?

File: scripts/app.js [hunk 3]
```diff
@@ -11,6 +14,13 @@ const players = [
   },
 ];
 
+let gameData = [
```
Comment (johnbradshw) [inline scripts/app.js]: If the variables is only used in either `config.js` or `game.js` then it's best to define it in those JS files (applies to all variables defined here)

**Review by johnbradshw — DISMISSED** [review]

File: scripts/game.js (hunk 2 above)
Comment (jacquelyn-bradshaw) [reply scripts/game.js]: == only compares values and === compares type and value

Comment (jacquelyn-bradshaw) [reply scripts/game.js]: No

File: scripts/app.js (hunk 3 above)
Comment (jacquelyn-bradshaw) [reply scripts/app.js]: fixed

File: scripts/game.js (hunk 2 above)
Comment (johnbradshw) [reply scripts/game.js]: e.g.  `activePlayer = activePlayer === 0 ? 1 : 0`

**Review by johnbradshw — DISMISSED** [review]

**Review by johnbradshw — DISMISSED** [review]

**Review by johnbradshw — APPROVED** [review]

**Pull request merged**